compose/config/validation.py

```diff
@@ -64,6 +90,26 @@ def format_expose(instance):
     return True
 
 
+@FormatChecker.cls_checks("subnet_ip_address", raises=ValidationError)
+def format_subnet_ip_address(instance):
+    if isinstance(instance, six.string_types):
+        if '/' not in instance:
+            raise ValidationError("should be of the format 'IP_ADDRESS/CIDR'")
+
+        ip_address, cidr = instance.split('/')
+
+        if re.match(VALID_REGEX_IPV4_ADDR, ip_address):
+            if not re.match(VALID_REGEX_IPV4_CIDR, cidr):
```

Comment:
I don't think we need a regexp for this part - a simple int conversion should suffice.

Reply:
Would it be okay if I would just include the CIDR prefix check within the regex pattern then, as all CIDRs would require that prefix?

Reply:
That would work too.